### PR TITLE
Add code of conduct page

### PR DESCRIPTION
**Pull request by paulodiovani**
- Add page
- Add to header menu
- Add _Início_ link to header menu
- Add also in CODE_OF_CONDUCT.md according to github patterns

screenshot from 2017-09-06 15-01-30

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+sexualizado.
+
+Se um participante se envolve em comportamento de assédio, os organizadores da
+conferência podem tomar todas as medidas que considerem adequadas, incluindo
+avisar o ofensor ou expulsá-lo da conferência sem nenhum reembolso.
+
+Caso seja assediado, perceba que alguém está sendo assediado ou tenha alguma
+dúvida, entre em contato com um membro da equipe da conferência imediatamente.
+Membros da equipe da conferência podem ser identificados pois estarão vestindo
+camisetas marcadas, ou com os devidos crachás de staff.
```
Comment (boniattirodrigo) [inline CODE_OF_CONDUCT.md]: Will we be?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Não tome atitudes por conta própria e denuncie a situação de forma discreta, a
+fim de mantermos a segurança de todos, a ordem, e o sigilo.
+
+A equipe da conferência estará disposta a auxiliar os participantes a contatarem
+a segurança do hotel ou local, assim como as devidas aplicações da lei local,
```
Comment (boniattirodrigo) [inline CODE_OF_CONDUCT.md]: hotel?!

**Comment by boniattirodrigo** [comment]
The header has a lot of links, I guess we could remove "Início" and change the name of "Chamada para trabalhos".

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (paulodiovani) [reply CODE_OF_CONDUCT.md]: I don't think so. Changing.

**Comment by paulodiovani** [comment]
The _Início_ is the only link to reach the top of the page again. I can remove if you think it's unecessary.

_Chamada para Trabalhos_ is necessary.

**Pull request merged**